src/lib/serviceRoutines/postUpdateContext.cpp

```diff
@@ -124,7 +124,7 @@ static bool updateForward
   {
     std::string details = std::string("invalid providing application '") + upcrP->contextProvider + "'";
 
-    alarmMgr.badInput(clientIp, details);
+    alarmMgr.badInput(clientIp, details, upcrP->contextProvider);
```

Comment:
`upcrP->contextProvider` is part of details string, should be added?

Reply:
Fixed in 10cdfce2c